driver/normalizer/normalizer.go

```diff
@@ -74,16 +74,22 @@ var Preprocessors = []Mapping{
 
 // Normalizers is the main block of normalization rules to convert native AST to semantic UAST.
 var Normalizers = []Mapping{
+	//FIXME(bzz): save all 3 in new parent Group \w ordered comments
+	Map(
+		Part("_", Fields{
+			{Name: "leadingComments", Drop: true, Op: Check(OfKind(nodes.KindNil | nodes.KindArray), Empty(Any()))},
```

Comment:
It should be okay like this:
```suggestion
			{Name: "leadingComments", Drop: true, Op: Any()},
```
We can't save them for now anyway.

Reply:
Hm, are you 

sure? That's what I tried at first as well, but than if we do that - many fixtures will change and every comment is going to be left only once, on the top level `comments:[..]` instead of how it's done now when we have them also in-place e.g https://github.com/bblfsh/javascript-driver/blob/master/fixtures/comment.js.uast#L343